hugr-passes/Cargo.toml

```
@@ -0,0 +1,17 @@
+[package]
+name = "hugr-passes"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+hugr = { path = "../hugr" }
```

```suggestion
hugr = { path = "../hugr", version = "0.4.0 }
```
crates.io requires published packages to depend only on other published packages.

Cargo will use the path for local development but ignore it when publishing.